schema_salad/main.py

```diff
@@ -105,6 +105,21 @@ def main(argsl=None):  # type: (Optional[List[str]]) -> int
         help="Generate classes in target language, currently supported: python",
     )
 
+    parser.add_argument(
+        "--codegen-target",
+        type=str,
+        default=None,
+        help="Defaults to sys.stdout for python (and ./ for hidden Java codegen)",
```

Comment:
Hidden?

Reply:
"Generate classes in target language, currently supported: python"

I'll add java to the list and remove the word hidden in a follow up.